.changeset/cold-points-greet.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sigstore/mock': minor
+'@sigstore/cli': minor
+---
+
+Drop node 14 support
```

Comment:
Since these two packages haven't yet reached a stable 1.0 release, I'm not going to treat the node 16 requirement as a major change.